Gemfile.lock

```diff
@@ -2,6 +2,7 @@ PATH
   remote: .
   specs:
     commons-builder (0.1.0)
+      liquid
```

Comment:
It's a very small thing, so I wouldn't worry about changing this now, but for the future when adding a dependency, I quite like to have the changes to Gemfile and Gemfile.lock in a separate commit with a message saying what you're adding and why, since it keeps the more substantive commit smaller.

Reply:
I've pulled this out into a separate commit anyway, which gives an opportunity to explain the choice of liquid.